spec/Data_Parallelism.tex

```diff
@@ -280,9 +284,110 @@ \section{Forall Intents}
 \end{rationale}
 
 
+\section{Task-Private Variables}
+\label{Task_Private_Variables}
+\index{task-private variables}
+\index{shadow variables}
+\index{data parallelism!task-private variables}
+\index{data parallelism!shadow variables}
+\index{statements!forall@\chpl{forall}!task-private variables}
+\index{statements!forall@\chpl{forall}!shadow variables}
+
+A \emph{task-private variable} declared in a forall loop results
+in a separate shadow variable in each task created by the forall
+loop's parallel iterator, as well as a separate shadow variable
+created at the top level of the parallel iterator itself.
+
+The shadow variable is created at the start of the task or the
+iterator and destroyed at the end of the task/iterator.
```

Comment:
I think you can just say 'task' and not mention iterator here, since we already state in the previous paragraph that forall loops create tasks.